net/hass/files/hassd.sh

```diff
@@ -0,0 +1,29 @@
+#!/bin/sh
+# todo: hostapd white (black) list so we don't have to listen to all APs
+
+source "/lib/functions.sh"
+config_load hass
+
+logger -t $0 -p info "Starting up"
+
+source /usr/lib/hass/functions.sh
```

Comment:
you've "" quoted above, but not here.  I don't care which you use, but consistency is nice.

Reply:
you're right, just copied the upper part everywhere.